digi/xbee/models/atcomm.py

```diff
@@ -35,7 +35,9 @@ class ATStringCommand(Enum):
     AR = ("AR", "Many-to-one route broadcast time")
     AS = ("AS", "Active scan")
     BD = ("BD", "UART baudrate")
+    BI = ("BI", "Bluetooth identifier")
     BL = ("BL", "Bluetooth address")
+    BP = ("BP", "Bluetooth advertisement power")
```

Comment:
Just to be in line with the documentation from the firmware, I would update the description of this command to: `Bluetooth advertisements power level`